engine/Assets/Scripts/Modes/MatchMode/MatchMode.cs

```diff
@@ -11,8 +11,8 @@ namespace Modes.MatchMode {
     public class MatchMode : IMode {
         public static MatchResultsTracker MatchResultsTracker;
 
-        public static int CurrentFieldIndex = -1;
-        public static int[] SelectedRobots  = new int[6];
+        /// Integers to represent which robots the user selected in the MatchModeModal
+        public static int[] SelectedRobots = new int[6];
```

Comment:
I'll change this to actually use the Miralive files when I add in optimizations.